docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -303,14 +365,18 @@ To remove all hypothetical indexes, log out or quit your session.
 
 For more information, refer to the [HypoPG documentation](https://hypopg.readthedocs.io/en/rel1_stable/).
 
-### passwordcheck example
+### passwordcheck
```

Comment:
HypoPG is missing an intro : https://deploy-preview-17664--infallible-bardeen-164bc9.netlify.app/preview/explore/ysql-language-features/pg-extensions/#hypopg-example